mne_connectivity/envelope.py

```diff
@@ -159,33 +151,32 @@ def envelope_correlation(data, indices=None, names=None, combine='mean',
 
     # apply function on correlation structure
     n_epochs = len(corrs)
-    corr = fun(corrs)
 
-    if combine is None:
-        # ravel from 2D connectivity into 1D array
-        # over all epochs
-        corr = np.array([_corr.flatten() for _corr in corr])
-    else:
-        # ravel N x N array into 1D array
-        corr = corr.flatten()
+    # ravel from 2D connectivity into 1D array
+    # over all epochs
+    corr = np.array([_corr.flatten() for _corr in corrs])
 
     # create the connectivity container
     times = None
 
-    # create epochs axis
-    if combine is not None:
-        corr = corr[np.newaxis, ...]
-
     # create time axis
     corr = corr[..., np.newaxis]
 
-    print(corr.shape)
+    if indices is None:
+        indices = 'symmetric'
+
+    # only get the upper-triu indices
+    triu_inds = np.triu_indices(n_nodes, k=0)
+    raveled_triu_inds = np.ravel_multi_index(
+        triu_inds, dims=(n_nodes, n_nodes))
+    corr = corr[:, raveled_triu_inds, ...]
+
     conn = EpochTemporalConnectivity(
         data=corr,
         names=names,
         times=times,
         method='envelope correlation',
-        indices=indices,
+        indices='symmetric',
```

Comment:
Why do you set it explicitly here when it is only set to `'symmetric'` if `indices is None` in line 166? If output is always symmetric, are line 165-166 even needed?

Reply:
I was thinking of enabling `indices` to allow for a subset to be computed, since any connectivity function that is a bivariate function `f(signal_1, signal_2)` can allow for `indices` to define a subset.

Then this line would be needed since the default would be "all-to-all symmetric". WDYT?

Reply:
Ah, so you are anticipating future functionality?

Reply:
Yep! 

Well... now that you bring it up, I probably should've just implemented it first. But yeah figured this is the way the code will look like once I add lines to account for `indices = ([...], [...])`.

Reply:
Going to try adding now that I've addressed the other comments.

Reply:
Actually, upon re-reading the code, idk if that's worth doing. 

Envelope correlation is already very easy to get, so idk if it's worth adding the `indices` unless it comes up.

Reply:
Going to table this and move onwards w/ other connectivity functions and examples. I figure we can add these niceties if ppl really want them later on.

Reply:
Sounds good!